From 7d0db7150a6dd508212625a48bcad31b5c3daad6 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 24 Aug 2020 14:26:26 +0800
Subject: [PATCH] Test case4 for Change guid value in Protocols

Description: Change guid value in Protocols section (to gEfiLoadFileProtocolGuid = { 0x56E24071, 0x96DC, 0x11D2, { 0x8E, 0x3E, 0x00, 0xA0, 0xC9, 0x69, 0x82, 0x4F }}) in MdeModulePkg/MdeModulePkg.dec.

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdePkg/MdePkg.dec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdePkg/MdePkg.dec b/MdePkg/MdePkg.dec
index 682e61cb88..84ace7ca44 100644
--- a/MdePkg/MdePkg.dec
+++ b/MdePkg/MdePkg.dec
@@ -1425,7 +1425,7 @@
   gEfiLoadedImageDevicePathProtocolGuid = { 0xbc62157e, 0x3e33, 0x4fec, {0x99, 0x20, 0x2d, 0x3b, 0x36, 0xd7, 0x50, 0xdf }}
 
   ## Include/Protocol/LoadFile.h
-  gEfiLoadFileProtocolGuid       = { 0x56EC3091, 0x954C, 0x11D2, { 0x8E, 0x3F, 0x00, 0xA0, 0xC9, 0x69, 0x72, 0x3B }}
+  gEfiLoadFileProtocolGuid       = { 0x56E24071, 0x96DC, 0x11D2, { 0x8E, 0x3E, 0x00, 0xA0, 0xC9, 0x69, 0x82, 0x4F }}
 
   ## Include/Protocol/LoadFile2.h
   gEfiLoadFile2ProtocolGuid       = { 0x4006c0c1, 0xfcb3, 0x403e, {0x99, 0x6d, 0x4a, 0x6c, 0x87, 0x24, 0xe0, 0x6d }}
-- 
2.27.0.windows.1

